lib/dispatcher.rb

```diff
@@ -29,15 +29,15 @@ def email_sent_confirmation(proposal)
 
   def deliver_new_proposal_emails(proposal)
     proposal.currently_awaiting_approvals.each do |approval|
-      self.email_approver(approval)
+      email_approver(approval)
     end
-    self.email_observers(proposal)
-    self.email_sent_confirmation(proposal)
+
+    email_observers(proposal)
+    email_sent_confirmation(proposal)
   end
 
   def deliver_attachment_emails(proposal)
     proposal.users.each do |user|
-      # do not send email to approvers who have not yet heard about the proposal
```

Comment:
added some descriptive private methods here 1115555e27d7566f8f893d71b7962d765b455e35

I think this is actually a win because the previous comment mentioned not sending emails to approvers who don't know about a proposal, but didn't mention that we also want to send emails to all non-approvers. methods make that clearer, I think